src/overlays/actors/ovl_Bg_Iknin_Susceil/z_bg_iknin_susceil.c

```diff
@@ -21,46 +35,228 @@ const ActorInit Bg_Iknin_Susceil_InitVars = {
     (ActorFunc)BgIkninSusceil_Update,
     (ActorFunc)BgIkninSusceil_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Bg_Iknin_Susceil_0x80C0A740/func_80C0A740.asm")
+s32 unused = 0;
+f32 D_80C0B0E4 = 960.0f;
+Vec2f D_80C0B0E8 = { -320.0f, 0.0f };
+s8 D_80C0B0F0[] = { 0x00, 0x00, 0x07, 0x0A, 0x0A, 0x0B, 0x0B, 0x00 };
+s8 D_80C0B0F8[] = { 0x01, 0x02, 0x00, 0x01, 0x02, 0x01, 0x02, 0x00 };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+s32 func_80C0A740(BgIkninSusceil* this, GlobalContext* globalCtx) {
+    s32 pad2[2];
+    Vec3f offset;
+    ActorPlayer* player = PLAYER;
+
+    Actor_CalcOffsetOrientedToDrawRotation(&this->dyna.actor, &offset, &player->base.world.pos);
+
+    return ((D_80C0B0E8.x < offset.z) && (offset.z < D_80C0B0E8.y) && (offset.x > -240.0f) && (offset.x < D_80C0B0E4));
+}
+
+void func_80C0A804(BgIkninSusceil* this, GlobalContext* globalCtx) {
+    func_800C6314(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
+}
+
+void func_80C0A838(BgIkninSusceil* this, GlobalContext* globalCtx) {
+    func_800C62BC(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
+}
+
+void func_80C0A86C(BgIkninSusceil* this, GlobalContext* globalCtx, s16 y, s16 countdown, s32 arg4) {
+    s32 pad;
+    s16 quake = Quake_Add(ACTIVE_CAM, 3);
+
+    Quake_SetSpeed(quake, 0x7B30);
+    Quake_SetQuakeValues(quake, y, 0, 0, 0);
+    Quake_SetCountdown(quake, countdown);
+    if (arg4 == 1) {
+        func_8013ECE0(10000.0f, 255, 20, 150);
+        return;
+    }
+    if (arg4 == 2) {
+        func_8013ECE0(10000.0f, 180, 20, 100);
+        return;
+    }
+    if (arg4 == 3) {
+        func_8013ECE0(10000.0f, 120, 20, 10);
+    }
+}
+
+s32 func_80C0A95C(BgIkninSusceil* this, GlobalContext* globalCtx) {
+    s32 phi_t0 = true;
+    s32 i;
+    f32 new_var;
+    ActorPlayer* player = PLAYER;
+    Vec3f offset;
+    f32 temp1, temp2, temp3, temp4;
+
+    Actor_CalcOffsetOrientedToDrawRotation(&this->dyna.actor, &offset, &player->base.world.pos);
+    for (i = 0; i < 7; i++) {
+        temp3 = (D_80C0B0F0[i] * 80.0f) + 0.5f;
+        temp4 = (D_80C0B0F0[i] * 80.0f) + 79.5f;
+        if (1) {}
+        new_var = D_80C0B0F8[i] * (-80.0f); // temp seems to actually be needed for matching
```

Comment:
Are the parentheses needed?
```suggestion
        new_var = D_80C0B0F8[i] * -80.0f; // temp seems to actually be needed for matching
```